Mathlib/Analysis/SpecialFunctions/ImproperIntegrals.lean

```diff
@@ -51,6 +58,54 @@ theorem integral_exp_neg_Ioi (c : ℝ) : (∫ x : ℝ in Ioi c, exp (-x)) = exp
 theorem integral_exp_neg_Ioi_zero : (∫ x : ℝ in Ioi 0, exp (-x)) = 1 := by
   simpa only [neg_zero, exp_zero] using integral_exp_neg_Ioi 0
 
+theorem integrableOn_exp_mul_complex_Ioi {a : ℂ} (ha : a.re < 0) (c : ℝ) :
+    IntegrableOn (fun x : ℝ => Complex.exp (a * x)) (Ioi c) := by
+  refine (integrable_norm_iff ?_).mp ?_
+  · apply Continuous.aestronglyMeasurable
+    fun_prop
+  · conv in (fun x => _) =>
+      ext x
+      rw [(by simp [Complex.norm_exp] : ‖Complex.exp (a * x)‖ = exp (-(-a.re * x)))]
+    have a_neg : 0 < -a.re := by simp [ha]
+    apply (integrableOn_Ioi_comp_mul_left_iff (fun x => exp (-x)) c a_neg).mpr
+    apply integrableOn_exp_neg_Ioi
+
+theorem integrableOn_exp_mul_complex_Iic {a : ℂ} (ha : 0 < a.re) (c : ℝ) :
+    IntegrableOn (fun x : ℝ => Complex.exp (a * x)) (Iic c) := by
```

Comment:
This should follow from the above lemma and the `integrableOn.comp_neg` lemma I mentioned.